src/components/ButtonSystem/Storybook/ButtonSystem.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ButtonSystem from './ButtonSystem';
+
+export default {
+  title: 'Button System/All',
```

Comment:
I would probably just call it "Buttons", and add more stories for some additional cases, such as the button group and icon-only button